src/components/Tipseen/Tipseen.tsx

```diff
@@ -1,46 +1,83 @@
-import { forwardRef, Fragment, useMemo, useRef } from "react";
-import PropTypes from "prop-types";
+import { forwardRef, Fragment, ReactElement, useMemo, useRef } from "react";
+import { DialogPosition } from "../../constants/positions";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import Tooltip from "../../components/Tooltip/Tooltip";
 import Button from "../../components/Button/Button";
 import { BEMClass } from "../../helpers/bem-helper";
 import Icon from "../../components/Icon/Icon";
 import CloseSmall from "../../components/Icon/Icons/components/CloseSmall";
-import { DialogPosition } from "../../constants/positions";
-import { AnimationType, JustifyType } from "../../constants/dialog";
+import { AnimationType, EMPTY_ARR, HideShowEvent, JustifyType } from "../../constants";
 import TipseenTitle from "./TipseenTitle";
 import { TIPSEEN_CLOSE_BUTTON_ARIA_LABEL } from "./TipseenConstants";
+import { VibeComponent, VibeComponentProps } from "../../types";
+import { MoveBy } from "../../types/MoveBy";
+import { ElementContent } from "../../types/ElementContent";
+import { Modifier } from "react-popper";
 import styles from "./Tipseen.module.scss";
 
 const TIPSEEN_BASE_CSS_CLASS = "monday-style-tipseen";
 const bemHelper = BEMClass(TIPSEEN_BASE_CSS_CLASS);
 
-const Tipseen = forwardRef(
+interface TipseenProps extends VibeComponentProps {
+  position?: DialogPosition;
+  animationType?: AnimationType;
+  hideDelay?: number;
+  showDelay?: number;
+  title?: string;
+  isCloseButtonHidden?: boolean;
+  children?: ReactElement;
+  containerSelector?: string;
+  hideTrigger?: HideShowEvent | Array<HideShowEvent>;
+  showTrigger?: HideShowEvent | Array<HideShowEvent>;
+  justify?: JustifyType;
+  width?: number;
+  moveBy?: MoveBy;
+  hideWhenReferenceHidden?: boolean;
+  /**
+   * when false, the arrow of the tooltip is hidden
+   */
+  tip?: boolean;
+  /**
+   * PopperJS Modifiers type
+   * https://popper.js.org/docs/v2/modifiers/
+   */
+  modifiers?: Array<Modifier<any>>;
+  closeAriaLabel?: string;
+  isCloseButtonOnImage?: boolean;
+  onClose?: () => void;
+  content?: ElementContent;
+}
+
+const Tipseen: VibeComponent<TipseenProps> & {
+  positions?: typeof DialogPosition;
+  animationTypes?: AnimationType;
+  justifyTypes?: JustifyType;
+} = forwardRef(
   (
     {
       className,
```

Comment:
regarding className default value. we had issues in the past when changed empty string to undefined in cases that some conditions in the code checked if the prop is equal to undefined and then changed it cause change of behavior.
Please make sure change here the prop will not casue the same effect or change the default value to be empty string like before :)